UI/MapUI.gd

```diff
@@ -13,16 +13,15 @@ var balloon_scene: PackedScene = load("res://Dialog/EventDialogueWindow.tscn")
 var test_dialog: DialogueResource = load("res://Dialog/test.dialogue")
 
 var _padding_offset: int = 20
-var _MINIMUM_ROOM_WIDTH: int = 510
-var _MINIMUM_ROOM_HEIGHT: int = 490
+var _MINIMUM_ROOM_WIDTH: int = 910
+var _MINIMUM_ROOM_HEIGHT: int = 890
```

Comment:
actually, isn't that the minimum map width and height ? because room_width and height makes it sound that a single room is at least 910x890
Regarding this, I would make the map a bit bigger than what we already have. The rooms are dynamically sized to fit inside the map container, so a bigger map would give us bigger rooms, which might be better than to click on small rooms